Libraries/LibWeb/MediaCapabilitiesAPI/MediaCapabilities.h

```diff
@@ -18,10 +107,27 @@ class MediaCapabilities final : public Bindings::PlatformObject {
     static GC::Ref<MediaCapabilities> create(JS::Realm&);
     virtual ~MediaCapabilities() override = default;
 
+    // https://w3c.github.io/media-capabilities/#queue-a-media-capabilities-task
+    void queue_a_media_capabilities_task(Function<void()>);
+
+    // https://w3c.github.io/media-capabilities/#dom-mediacapabilities-decodinginfo
+    GC::Ref<WebIDL::Promise> decoding_info(MediaDecodingConfiguration const& configuration);
```

Comment:
nit for this function, and the other two below, we can leave `configuration` variable name out of the declaration since it is clear from the type itself.